index.html

```diff
@@ -339,73 +340,62 @@ <h3>Methods</h3>
-      by adding an additional internal slot and attribute for the playoutDelay.
+      by adding an additional attribute to adjust the receiver's jitter buffer.
     </p>
-    <section id="rtcrtpreceiver-playoutdelay-modifications">
+    <section id="rtcrtpreceiver-jitterbufferdepth-modifications">
       <h2>Modification to existing procedures</h2>
       <p>
-        Let {{RTCRtpReceiver}} objects have a <dfn data-dfn-for=RTCRtpReceiver>[[\PlayoutDelay]]</dfn> internal slot initially
+        Let {{RTCRtpReceiver}} objects have a <dfn data-dfn-for=RTCRtpReceiver>[[\JitterBufferDepth]]</dfn> internal slot initially
         initialized to <code>null</code>.
       </p>
     </section>
-    <section id="rtcrtpreceiver-playoutdelay-rtcrtpreceiver-interface">
+    <section id="rtcrtpreceiver-jitterbufferdepth-rtcrtpreceiver-interface">
       <h2>{{RTCRtpReceiver}} interface extensions</h2>
       <pre class="idl">partial interface RTCRtpReceiver {
-  attribute double? playoutDelay;
+  attribute double? jitterBufferDepth;
 };</pre>
     </section>
-    <section id="rtcrtpreceiver-playoutdelay-attributes">
+    <section id="rtcrtpreceiver-jitterbufferdepth-attributes">
       <h2>Attributes</h2>
       <dl data-link-for="RTCRtpReceiver" data-dfn-for="RTCRtpReceiver"
       class="attributes">
-        <dt><dfn>playoutDelay</dfn> of type <span
+        <dt><dfn>jitterBufferDepth</dfn> of type <span
           class="idlAttrType">double</span>, nullable</dt>
         <dd>
           <p>This attribute allows the application to specify a target duration
-          of time between network reception of media and playout. The
-          <a>user agent</a> SHOULD NOT playout audio or video that is received unless
-          this amount of time has passed in seconds, allowing the <a>user agent</a>
-          to perform more or less buffering than it might otherwise do.
-          This allows to influence the tradeoffs between having a higher delay
-          and the risk that buffers such as the jitter buffer will run out of
-          audio or video frames to play due to network jitter.</p>
-          <p>The <a>user agent</a> may have a <dfn>minimum allowed delay</dfn> and a
-          <dfn>maximum allowed delay</dfn> reflecting what the <a>user agent</a> is
+          of time in milliseconds of media for the {{RTCRtpReceiver}}'s jitter
+          buffer to hold. This influences the amount of buffering done by the
+          <a>user agent</a>, and allows applications to control the tradeoff between
+          playout delay and the risk of running out of audio or video frames due to
+          network jitter.
+
+          <p>The <a>user agent</a> MUST have a <dfn>minimum allowed depth</dfn> and a
+          <dfn>maximum allowed depth</dfn> reflecting what the <a>user agent</a> is
           able or willing to provide based on network conditions and memory
           constraints.</p>
```

Comment:
```suggestion
          constraints, and may change at any time.</p>
```

Reply:
Just to clarify that these are not constant min and max values, if that wasn't already obvious

Reply:
Ah, that was not clear before, so thanks for clarifying! We should avoid MAY here then probably 
```suggestion
          constraints, which can change at any time.</p>
```

Reply:
sgtm